### ISSUE TITLE
Active Forging Wallet Deposits

### ISSUE THREAD

**Issue by joesixpack** [metahashorg/metagate#18]
For deposits sent to the Active Forging Wallet by mistake, it would be nice to enable receiving tickets for those deposits.  It was never made too clear not to use the Active Forging Wallet for deposits, even though that is the default wallet that shows up.

**Comment by davidbios** [comment]
I agree, I have made same problem then I make my own wallet again to use it
